README.md

```diff
@@ -48,6 +49,52 @@ mocha --grep '"specificProperty" test name' ./tests/10-specific-test-suite.js
 These test suites use tags to identify which implementation's endpoints are used in tests.
 If you need to change the tag the suites will run on you can change it in `./config/runner.json`.
 
+### Testing Locally
+
+If you want to test implementations or just endpoints running locally, you can
+copy `localImplementationsConfig.cjs` to `.localImplementationsConfig.cjs`
+in the root directory of the test suite.
+
+```bash
+cp localImplementationsConfig.cjs .localImplementationsConfig.cjs
+```
+
+Git is set to ignore `.localImplementationsConfig.cjs` by default.
+
+This file must be a CommonJS module that exports an array of implementations:
+
+```js
+// .localImplementationsConfig.cjs defining local implementations
+// you can specify a BASE_URL before running the tests such as:
+// BASE_URL=http://localhost:40443/zDdfsdfs npm test
+const baseUrl = process.env.BASE_URL || 'https://localhost:40443/id';
+module.exports = [{
+  name: 'My Company',
+  implementation: 'My Implementation Name',
+  issuers: [{
+    id: 'did:myMethod:implementation:issuer:id',
+    endpoint: `${baseUrl}/credentials/issue`,
+    tags: ['eddsa-rdfc-2022', 'localhost']
+  }],
+  verifiers: [{
+    id: 'did:myMethod:implementation:verifier:id',
+    endpoint: `${baseUrl}/credentials/verify`,
+    tags: ['eddsa-rdfc-2022', 'localhost']
+  }]
+}];
+```
+
+After adding the config file, both the localhost implementations and other
+implementations matching the test tag will be included in the test run.
+
+To specifically test only the localhost implementation, modify the test suite to
+filter implementations based on a specific tag in your local configuration file.
+
+For instance, if your `.localImplementationsConfig.cjs` config file looks like
+the config above, you can adjust the tag used in each test suite
```

Comment:
Is this adjustment made in code? I don't quite follow what the expected action is supposed to be here.

Reply:
The next line actually clarified you should modify `./config/runner.json`, but I went ahead and merged the two sentences into one here: https://github.com/w3c/vc-di-eddsa-test-suite/pull/39/commits/e511dde18534a9787ff6c3e9941933e2081ade9f

LMK if that helps. You should be modifying a json manifest file.